### PR TITLE
Added compatibility with Symfony 3.3.13

### LINKED ISSUE #78: Resolve FileType BC break
Introduced in [24993](https://github.com/symfony/symfony/pull/24993). The solution is to create a form extension for AbstractType, which would overwrite the RequestHandler. The issue is with the `isFileUpload()` method.

### PR DESCRIPTION
**Pull request by chives**
Resolves #78

### CODE REVIEW

**Comment by chives** [comment]
@fsi-open

File: Form/TypeExtension/FileUploadFormExtension.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    {
+        $this->configureOptions($resolver);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefault('fsi_request_handler', true);
```
Comment (szymach) [inline Form/TypeExtension/FileUploadFormExtension.php]: What is this variable for?

Comment (szymach) [reply Form/TypeExtension/FileUploadFormExtension.php]: Also, where them test cases at? :)

**Comment by chives** [comment]
@fsi-open

**Pull request merged**